lib/uinta/plug.ex

```diff
@@ -275,5 +287,18 @@ if Code.ensure_loaded?(Plug) do
     defp query_type("mutation" <> _), do: "MUTATION"
     defp query_type("{" <> _), do: "QUERY"
     defp query_type(_), do: nil
+
+    defp should_log?(ratio) do
```

Comment:
What would you think of the name `should_include_in_sample?` or something to make it more obvious from the high level what this is checking?